### ISSUE TITLE
Default Battle Menu Background Displayed Incorrectly in 16:9

### ISSUE THREAD

**Issue by St0rmFlag** [shaddatic/sa2b-render-fix#118]
I made a request for this on GB a while back but it was never delivered, so I thought this might be something that falls within the scope of your mod.

Basically the background for the default Battle menu theme doesn't display correctly in 16:9. Unlike the character backgrounds, which are stretched to fit the screen, the default one is incredibly zoomed in a cut off at the bottom. This isn't the case when playing in 4:3, or widescreen stretched. 

16:9
![Screenshot (6)](https://github.com/user-attachments/assets/b982ca6c-33cb-4063-9d75-422008df11cf)

4:3
![Screenshot (7)](https://github.com/user-attachments/assets/8d61dc7a-6447-4fd3-86e5-45a22f9e9b35)

**Comment by St0rmFlag** [comment]
Letting you know I was able to fix this. The files that handle the 16:9 backgrounds are called _screenEffect_DLC_0/1_ in the SOC folder. I plan to release a mod that uses a higher quality version of the bg made by Speeps.

![Image](https://github.com/user-attachments/assets/5e483dba-2270-4c7e-9894-1f9a27152ea1)

**Comment by shaddatic** [comment]
Yeah, I don't think I'm going to implement stretching into RF like this. I'll tag it as an enhancement, but it's very unlikely I do this
